src/styles.scss

```diff
@@ -257,9 +257,29 @@
   }
 
   .ClaimableBalances {
-    .Table {
-      --table-min-width: 500px;
-      width: auto;
+    &__header {
+      margin-bottom: 2rem;
+
+      span {
+        display: block;
+        margin-bottom: 1rem;
+      }
+
+      @media (min-width: 900px) {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        h2 {
+          margin-bottom: 0;
+        }
+
+        span {
+          display: inline-block;
+          margin-right: 0.5rem;
+          margin-bottom: 0;
+        }
+      }
```

Comment:
What's the idea here? Ideally, the design system components should "just" work. It's possible that there is a gap somewhere and it's not handling this use case.